project/Dependencies.scala

```diff
@@ -17,9 +17,12 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.9.0"
 
+  //TODO: revert
   private val workbenchLibsHash = "01a11c3"
-  val serviceTestV = s"2.1-$workbenchLibsHash"
-  val workbenchModelV = s"0.17-$workbenchLibsHash"
+  val serviceTestV = s"3.1-7dbb965"
```

Comment:
I need this version for the azure tests... hoping it doesn't break the GCP tests with the selenium issue 😬